src/nomad_simulations/schema_packages/model_system.py

```diff
@@ -1033,55 +1031,58 @@ class ModelSystem(System):
 
     def __init__(self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs):
         super().__init__(m_def, m_context, **kwargs)
-        self.__is_atomic_flag = None
+        self._cache: dict[str, Any] = {}
 
-    @property
-    def _is_atomic(self) -> bool:
+    def is_atomic(self) -> bool:
```

Comment:
we probably do not need this wrapper just access thr cache and return False by default

Reply:
sorry, I do not get it, can you send some pseudo-code?